### PR TITLE
Util stories

### PR DESCRIPTION
**Pull request by MethodenMann**
https://tocco.github.io/tocco-client/pr_util-stories

### CODE REVIEW

File: packages/app-extensions/src/notifier/notifier.stories.js [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+import {Provider} from 'react-redux'
+import {Button, ButtonGroup, Typography} from 'tocco-ui'
+import {withReadme} from 'storybook-readme'
+
+import Readme from './README.md'
+import notifier from './'
+import appFactory from '../appFactory'
+
+const longText = `Lorem ipsum dolor sit amet, at sed inermis intellegam scriptorem, usu facete apeirian ad.
+Sit et meliore intellegam. Mel cu maluisset philosophia, pri et habeo oportere. Vis in purto verear luptatum, has
```
Comment (khanlifts) [inline packages/app-extensions/src/notifier/notifier.stories.js]: I think you should indent line 14/15.

File: packages/app-extensions/src/notifier/notifier.stories.js [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+class Story extends React.Component {
+  constructor(props) {
+    super(props)
+    this.store = appFactory.createStore({}, undefined, {}, 'notifier')
+    notifier.addToStore(this.store, true)
+  }
+
+  info1 = () => {
+    this.store.dispatch(notifier.info(
```
Comment (khanlifts) [inline packages/app-extensions/src/notifier/notifier.stories.js]: Can be written in one line like info2.

File: packages/app-extensions/src/notifier/README.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+### Setup
+Firstly the notifier util has be added to the store:
+```javascript
+notifier.addToStore(store, accept)
+```
+
+The first parameter is the redux store. The store is needed to adding it reducers and add saga middleware.
+The second parameter (accept) determines whether the app is handling the notifier action itself or just emits the
+actions to the parent app. Latter also needs the `emmitAction` util added to the store.
```
Comment (khanlifts) [inline packages/app-extensions/src/notifier/README.md]: Same as in errorLogging README.

File: packages/app-extensions/src/notifier/notifier.stories.js (hunk 1 above)
Comment (MethodenMann) [reply packages/app-extensions/src/notifier/notifier.stories.js]: Thats a multi line string.

**Pull request merged**